terraform/eks/iam.tf

```diff
@@ -0,0 +1,186 @@
+#---
+# Provides a role for the cluster-autoscaler
+#
+# Note that the resource spec will result in a failure if we have
+# multiple ASGs. I would like to find a way to successfully pass
+# in a list.
+#---
+
+resource "aws_iam_role" "cluster_autoscaler" {
```

Comment:
Don't believe you need to create this role, the module actually creates that for you